analyses/fusion_filtering/02-fusion-filtering.R

```diff
@@ -170,6 +170,9 @@ fusion_filtering_QC<-function(standardFusioncalls=standardFusioncalls,readingFra
 # QC filter: artifact and read support
 QCFiltered<-fusion_filtering_QC(standardFusioncalls = standardFusioncalls,junctionReadCountFilter = junctionReadCountFilter,spanningFragCountFilter = spanningFragCountFilter,readingFrameFilter = readingFrameFilter,artifactFilter = artifactFilter,readthroughFilter = readthroughFilter)
 
+saveRDS(QCFiltered,paste0(opt$outputfile,"_QC_filtered.RDS"))
```

Comment:
This add doesn't result in a new file being committed to the repository. Is this leftover from development?

Reply:
yeah I'm saving this intermediate file in scratch, should I commit that file as well ?

Reply:
Ah, nope - if it's in `scratch` this makes sense! I forgot the output from this (`opt$outputfile`) is going into scratch.